lib/eventboss/cli.rb

```diff
@@ -36,7 +36,7 @@ def run
     private
 
     def boot_system
-      Dotenv.load
+      Dotenv.load if defined?(Dotenv)
```

Comment:
From what I see in dotenv change log `Ruby >= 3.0 and Rails >= 6.1 are now required. Lock dotenv to ~> 2.0 if you are using an outdated Ruby or Rails version.` is the most breaking change. There might be a lot of apps with older Rails. I think that other than that it will be safe to keep the `dotenv` dependency, but without specifying the version. Thanks to that we can avoid using `if defined`